app/models/comfy/cms/category.rb

```diff
@@ -8,8 +8,8 @@ class Comfy::Cms::Category < ActiveRecord::Base
   validates_presence_of :label, :title_en, :title_cy
   validates_uniqueness_of :label, :title_en, :title_cy, scope: :site_id
 
-  belongs_to :small_image, class: Comfy::Cms::File
-  belongs_to :large_image, class: Comfy::Cms::File
+  belongs_to :small_image, class_name: Comfy::Cms::File
```

Comment:
I think typically the class name is passed as a string when used as this option.